runtime/mandala/tests/integration_test.rs

```diff
@@ -1855,3 +1855,17 @@ fn currency_id_convert() {
 		);
 	});
 }
+
+#[test]
+fn parachain_subaccounts_are_unique() {
+	ExtBuilder::default().build().execute_with(|| {
+		let parachain: AccountId = ParachainInfo::parachain_id().into_account();
+		let relaychain: AccountId = Utility::derivative_account_id(ParachainInfo::get().into_account(), 0);
+		let account1: AccountId =
+			hex_literal::hex!["7061726164000000000000000000000000000000000000000000000000000000"].into();
+		let account2: AccountId =
+			hex_literal::hex!["00465d6ab005c2fd8c4e0bf22a60fe3ce5ff035072ec74679f4babb4c6f00833"].into();
```

Comment:
@xlc is this what you mean?

Reply:
Yes. Just need to change the name `relaychain` to `sub_account`